scripts/build/redhat/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "etcher-electron",
+  "productName": "Etcher",
+  "genericName": "Etcher",
+  "homepage": "https://etcher.io/",
+  "icon": "assets/icon.png",
```

Comment:
Maybe its just me, but the icon doesn't seem to be working on Fedora:

<img width="755" alt="screenshot 2017-05-18 16 41 00" src="https://cloud.githubusercontent.com/assets/2192773/26222659/e52346d0-3be8-11e7-97f9-c9eb4ceb1561.png">